src/Dfc.CourseDirectory.Web/appsettings.json

```diff
@@ -78,7 +78,7 @@
     "FindAddressesBaseUrl": "https://services.postcodeanywhere.co.uk/PostcodeAnywhere/Interactive/FindByPostcode/v1.00/json3.ws?",
     "RetrieveAddressBaseUrl": "https://services.postcodeanywhere.co.uk/PostcodeAnywhere/Interactive/RetrieveById/v1.30/json3.ws?"
   },
-  "ProviderAzureSearchIndexName": "provider",
+  "ProvidersAzureSearchIndexName": "providers",
```

Comment:
Why have you changed the name of the config variable?

Reply:
as per pr description:

> "Config key name changed to avoid being overridden by any existing production configuration."

Reply:
There are no existing production configurations for the key.

Reply:
reverted